identity/app/clients/DiscussionClient.scala

```diff
@@ -82,4 +94,26 @@ class DiscussionClient(wsClient: WSClient, config: conf.IdentityConfiguration)(i
       case e => Future.failed(new DiscussionApiServiceException(s"findProfileStats failure to find $userId", e))
     }
   }
+
+  /**
+    * Check if the user has at least one comment.
+    * This method simply requests a single comment, and checks if the response is non-empty.
+    * If it is, the user has at least one publicly visible comment, and their profile can be shown.
+    */
+  def profileHasAtLeastOneComment(userId: String): Future[Boolean] = {
+
+    val apiPath = s"/profile/$userId/comments?page=1&pageSize=1"
+
+    val discussionCommentsF = for {
```

Comment:
We should probably rename this val since it now contains a boolean indicating if there's comments, not the actual comments.
 Maybe something like `containsCommentsF` or `hasCommentsF` would be more appropriate ?

Reply:
Good catch!